client/src/pages/organizations/Show.tsx

```diff
@@ -578,14 +588,13 @@ const OrganizationShow = ({ pageDispatchers }: OrganizationShowProps) => {
                     label="Attachments"
                     component={FieldHelper.ReadonlyField}
                     humanValue={
-                      <div className="attachment-card-list">
-                        {organization.attachments.map(attachment => (
-                          <AttachmentCard
-                            key={attachment.uuid}
-                            attachment={attachment}
-                          />
-                        ))}
-                      </div>
+                      <AttachmentsDetailView
+                        attachments={attachments}
+                        updateAttachments={updateAttachments}
+                        relatedObjectType={Organization.relatedObjectType}
+                        relatedObjectUuid={values.uuid}
+                        allowEdit={canAdministrateOrg}
```

Comment:
This is not entirely correct; whether attachment edit is enabled also depends on `Settings.fields.attachment.restrictToAdmins` (see **organizations/Form.tsx**).